front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -290,24 +302,22 @@ const AdminProjectsProjectGeneral = () => {
           }
         }
 
-        const imagesToAddPromises = projectImages
-          .filter((file) => !file.remote)
-          .map((file) => {
-            if (latestProjectId) {
-              return addProjectImage(latestProjectId, file.base64);
-            }
+        const cardToAddPromise =
+          croppedProjectCardBase64 &&
```

Comment:
Could you add this in a variable with a good name to indicate what you're checking here?

Reply:
Isn't just having the croppedProjectCardBase64 enough of a check that there's a new image? The fact that you're only using this value in the function call, not the other two (folderCardImage and !folderCardImage.remote), also indicates that the other two are probably not needed.

Reply:
So we could do 

```
const cardImageToAddPromise =
          croppedProjectCardBase64 && latestProjectId
            ? addProjectImage(latestProjectId, croppedProjectCardBase64)
            : null;
```

instead of 


```
        const cardImageWasChanged =
          croppedProjectCardBase64 &&
          projectCardImage &&
          !projectCardImage.remote;
        const cardImageToAddPromise =
        croppedProjectCardBase64 && latestProjectId
            ? addProjectImage(latestProjectId, croppedProjectCardBase64)
            : null;
```